app/user/rest.py

```diff
@@ -37,15 +42,8 @@ def update_user(user_id):
         delete_model_user(user)
     else:
         status_code = 200
-        # TODO there has got to be a better way to do the next three lines
-        update_user, errors = user_schema.load(request.get_json())
-        if errors:
-            return jsonify(result="error", message=errors), 400
-        update_dict, errors = user_schema.dump(update_user)
-        # TODO FIX ME
-        # Remove update_service model which is added to db.session
         db.session.rollback()
-        save_model_user(user, update_dict=update_dict)
+        save_model_user(user, update_dict=request.get_json())
```

Comment:
Will this fail if you pass in extra variables via http call?
The extra values will get passed to the sql alchemy and an exception will be raised.
That was the reason for the load and dump calls?